clients/fides-consent/rollup.config.js

```diff
@@ -20,9 +20,9 @@ const generateConsentConfig = () => {
   );
 
   /**
-   * @type {import('../../types/config').Config}
+   * @type {import('privacy-center/types/config').Config}
    */
-  const privacyCenterConfig = require("../../config/config.json");
+  const privacyCenterConfig = require("privacy-center/config/config.json");
```

Comment:
Oof, this is some circular stuff... again, not strictly related to this change but evidence that we need to decouple this from the *build*.

Reply:
yes! This is what I was talking about on our call - not great to have this circular dep